src/state/opamGlobalState.mli

```diff
@@ -27,6 +27,9 @@ val all_installed: 'a global_state -> package_set
 
 val switches: 'a global_state -> switch list
 
+(* Fold over switches, using switch selections. Switch selection file
+   [switch-state] is loaded only as read-only, no further check given opam root
+   version is done. *)
```

Comment:
```suggestion
(** Fold over switches, using switch selections. Switch selection file
   [switch-state] is loaded only read-only; no further checks are done on the opam root
   version. *)
```
Do I follow that correctly that it doesn't check `$OPAMROOT/config` for opam-root-version, it just trusts that the switch-state is valid?